swap_meet/item.py

```diff
@@ -1,2 +1,12 @@
 class Item:
-    pass
+    
+    def __init__(self, category='', condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        return f"The condition of this item is {self.condition}"
```

Comment:
While our tests don't evaluate the condition descriptions for each rating, the README describes the expected behavior for this function more in-depth. 

In industry, we won't always have tests available to shape our program's functionality so definitely make sure you're checking the requirements in a written document (like our README.md).